reference/types/boolean.md

```diff
@@ -22,10 +22,20 @@ This exercise lists all the available actions of the main character in a given s
 - boolean operators
 - boolean operators precedence
 
+### Pac-man game logic
+
+This exercise looks at various game events and determine if they have occured by looking at various game states.  The implementation in Elixir teaches:
+
+- boolean values
+- boolean operators
+- boolean operators precedence
+
 #### Implementations
```

Comment:
It now looks like JS has implemented the pac-man logic exercise :) I think you need to create a separate `Implementations` section for the pac-man logic, and keep the existing one in the other exercise's section.

Reply:
Is there an example of this? I don't exactly know what you are asking me to do.

Reply:
Yep, see [this example](https://github.com/exercism/v3/blob/66578af1af85132cde11ba165e6b5e6185304499/reference/types/floating_point_number.md). What I'm asking for is to have the list implementations grouped under the actual exercise. The example will help hopefully.

Reply:
Ah! okok

Reply:
C'est complet